compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -12815,7 +12815,7 @@ private STNode parseDocumentationLine(STNode hashToken) {
             case 1:
                 STNode docElement = docElements.get(0);
                 if (docElement.kind == SyntaxKind.DOCUMENTATION_DESCRIPTION) {
-                    if (((STToken) docElement).text().startsWith("# Deprecated")) {
+                    if (((STToken) docElement).text().startsWith(" # Deprecated")) {
```

Comment:
IMO amount of leading whitespaces shouldn't matter.
Is it possible to send this from the lexer itself, as a token?